### PR TITLE
Update github ci/cd

### PR DESCRIPTION
**Pull request by davidsuba**
Signed-off-by: David Suba <david.suba@ifne.eu>

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -22,30 +22,56 @@ jobs:
...
+      - lint-chart
     steps:
     - name: Checkout
       uses: actions/checkout@v2
 
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        driver-opts: network=host
```
Comment (matofeder) [inline .github/workflows/ci.yaml]: remove network=host

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -22,30 +22,56 @@ jobs:
...
+  build-image-install-chart:
     runs-on: ubuntu-latest
     needs:
-    - lint-chart
-    strategy:
-      matrix:
-        k8s:
-        - v1.17.5
-        - v1.19.1
+      - lint-chart
```
Comment (matofeder) [inline .github/workflows/ci.yaml:33]: optional - enable v1.17.[5|12] install test

File: .github/workflows/release-image.yaml [hunk 3]
```diff
@@ -43,10 +41,9 @@ jobs:
...
         git config --global user.name "$GITHUB_ACTOR"
         git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
-
     - name: Tag branch
       uses: anothrNick/github-tag-action@1.26.0
       env:
         GITHUB_TOKEN: ${{ secrets.CICD_TOKEN }}
         CUSTOM_TAG: 'image-v${{ steps.version.outputs.version }}'
-        RELEASE_BRANCHES: main
+        RELEASE_BRANCHES: main
```
Comment (matofeder) [inline .github/workflows/release-image.yaml]: :snowman:

**Review by matofeder — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (davidsuba) [reply .github/workflows/ci.yaml:33]: for this kind version, only v1.17.11 is available

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (davidsuba) [reply .github/workflows/ci.yaml]: removed

**Review by matofeder — APPROVED** [review]

**Pull request merged**